### ISSUE TITLE
Anti-measles antibody positivity in CSF

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10864]
**Preferred term label:**

**Synonyms**
Elevated measles antibody titers in cerebrospinal fluid

**Definition (free text, please give PubMed ID)**
PMID: 30680706 Neither of these tests is specific or sensitive to the diagnosis of SSPE and may correlate poorly with the clinical stage of the disease. Dyken’s modified criteria, often the diagnostic tool used in medical reports, includes clinical history, **elevated CSF measles antibody titres** as major markers, typical EEG, increased CSF immunoglobulin G (IgG), brain biopsy, and special molecular diagnostic tests to identify measles virus mutated genome as minor markers.28 Typically, diagnosis of SSPE requires two major and
one minor criteria.

PMID: 32809508  Major Criteria
-  One of the major criteria is having either a typical or atypical presentation. The typical presentation is defined as either acute, rapid, subacute, or chronic progressive or chronic relapsing-remitting.
 -   The atypical presentation includes seizures, prolonged stage I, or unusual age at presentation. Another major criterion is **elevated anti-measles antibodies greater than or equal to 1:4 in the CSF** or 1:256 in the serum.

**Parent term (use hpo.jax.org/app)**
Abnormal CSF protein concentration HP:0025456

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Subacute sclerosing leukoencephalitis ORPHA:2806

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 

Adding new term: 
- ID: HP:6001179
- label: Positive CSF anti-measles antibody test
- parents: [Positive CSF infectious agent test (HP:0430123)](https://hpo.jax.org/app/browse/term/HP:0430123)
- pmid: PMID:30680706|PMID:32809508
- def: The presence of antibodies in the cerebrospinal fluid (CSF) that react against a component of measles virus.
- comment: This finding may be observed with diseases including subacute sclerosing leukoencephalitis.
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10864
- synonyms: Elevated measles antibody titer in cerebrospinal fluid(NONE)
